src/service/ComposeService.ts

```diff
@@ -200,6 +200,7 @@ export class ComposeService {
                         image: presetData.symbolServerImage,
                         command: serverCommand,
                         stop_signal: 'SIGINT',
+                        stop_grace_period: '300s',
```

Comment:
Should this be configurable from the custom preset file?

Reply:
Do you think someone would like to increase it even more? OK, I will add it to be configurable.